repos/system_upgrade/el7toel8/models/repositoriessetuptasks.py

```diff
@@ -6,6 +6,7 @@ class RepositoriesSetupTasks(Model):
     """
     Information about repositories that must be managed in order to complete upgrade process.
 
+    !!! THIS MODEL IS OBSOLETE. Use for example CustomTargetRepositories instead. !!!
```

Comment:
I do not agree with the idea this model is kind of obsoleted. I can imagine in future that based on checks, there will be case a specific repository needs to be enabled / disabled. I am fun of   *Tasks* suffix as this is kind of self explaining name we could use across many problems we would like to handle on one place. But the approach is questionable. Let's see what will be the way in future.